ios/Classes/TiMapAnnotationProxy.m

```diff
@@ -414,6 +414,114 @@ -(void)setImage:(id)image
 	}
 }
 
+-(void)setShowMarker:(id)marker
```

Comment:
Reviewing these after the `marker: {}` change has been discussed. Having just the following would probably ease it:
```objc
-(void)setMarker:(id)marker
{
  ENSURE_TYPE(marker, NSDictionary);

  id glyphText = [TiUtils stringValue:@"glyphText" properties:marker];
  [self replaceValue:glyphText forKey:@"glyphText" notification:NO];

  // Same for the other marker properties here ...

  [self setNeedsRefreshingWithSelection:YES];
}
```